WooCommerce/Classes/Authentication/Navigation Exceptions/NoWooErrorViewModel.swift

```diff
@@ -4,38 +4,37 @@ import Yosemite
 /// Configuration and actions for an ULErrorViewController,
 /// modelling an error when WooCommerce is not installed or activated.
 final class NoWooErrorViewModel: ULErrorViewModel {
-    private let siteURL: String
+    private let site: Site
     private let showsConnectedStores: Bool
-    private let showsInstallButton: Bool
     private let analytics: Analytics
     private let stores: StoresManager
     private let setupCompletionHandler: (Int64) -> Void
 
     private var storePickerCoordinator: StorePickerCoordinator?
 
-    init(siteURL: String?,
+    init(site: Site,
          showsConnectedStores: Bool,
-         showsInstallButton: Bool,
          analytics: Analytics = ServiceLocator.analytics,
          stores: StoresManager = ServiceLocator.stores,
          onSetupCompletion: @escaping (Int64) -> Void) {
-        self.siteURL = siteURL ?? Localization.yourSite
+        self.site = site
         self.showsConnectedStores = showsConnectedStores
-        self.showsInstallButton = showsInstallButton
         self.analytics = analytics
         self.stores = stores
         self.setupCompletionHandler = onSetupCompletion
     }
 
     // MARK: - Data and configuration
+    var title: String? { site.name }
```

Comment:
nit: could probably be set in `init` with `self.title = site.name`?
```suggestion
    let title: String?
```

Reply:
Updated in e3161c72d366b1077e353f879926b8f4744acccf 👍